crates/sui-indexer-alt/src/handlers/mod.rs

```diff
@@ -632,14 +638,21 @@ fn committer<H: Handler + 'static>(
                     poll.reset_immediately();
                 }
 
-                Some(indexed )= rx.recv(), if pending_rows < H::MAX_PENDING_SIZE => {
+                Some(indexed) = rx.recv(), if pending_rows < H::MAX_PENDING_SIZE => {
```

Comment:
the back pressure logic here seems off, if `pending_rows >= H::MAX_PENDING_SIZE` and there are values in `rx`, the next value from `rx` will be consumed and then ignored?

Reply:
I tried to write a quick test to verify, but seems that the committer back pressure test will be good to have TempDB supported first, I am happy to help with the TempDB support and committer tests if you have not started on that yet

Reply:
The core parts of the indexer are changing quite a bit at the moment, so I would hold off on that for just now, otherwise we'll end up conflicting with each other a lot, but we can talk about ways to get involved in the sync today!